app/Decsys/Models/Wordlist/WordlistRules.cs

```diff
@@ -0,0 +1,9 @@
+namespace Decsys.Models.Wordlist;
+
+public class WordlistRules
+{
+    public string Type { get; set; } = string.Empty;
+    public bool IsInclusionCriteria { get; set; }
+    public string TargetProperty { get; set; } = string.Empty;
+    public string Operator { get; set; } = string.Empty;
```

Comment:
I think this needs a `Value` property?

but it'll probably have to be a basic JSON type. Maybe Json Node or Element or something (I don't know what all the type options are off the top of my head!). `JObject` won't work since it might (probably will) be raw values like strings or numbers.

Reply:
probably want [`JToken`](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_Linq_JToken.htm) or `JProperty`